### PR TITLE
Updated document related to develop the rails demo app

### PR DESCRIPTION
**Pull request by vijay-veeranki-moj**
related to #765

### CODE REVIEW

File: source/documentation/deploying-an-app/multicontainer-app-deploy.md [hunk 1]
```diff
@@ -87,6 +87,28 @@ You should be able to view the application in your browser at:
...
+      kubectl apply --filename monitoring-grafana-dashboard.yaml --namespace [your namespace]
+      ```
+
+* **Custom Alerts** - Follow the [guide][custom-alert] on how to create Custom Alerts, make corresponding changes to [prometheus-app-alert.yaml][alert-prometheusrule] file and apply the PrometheusRule to your namespace in the kubernetes cluster, this will create custom alerts for your application.
+      
+      ```
+      kubectl apply --filename prometheus-app-alert.yaml --namespace [your namespace]
+      ```
+
+* **Cronjobs** - Follow the [guide] [cron-jobs] on creating a Cronjob, make corresponding changes to [cronjob-ecr.yaml][cronjob-yaml] file and apply the Cronjob to your namespace in the kubernetes cluster, this runs a job periodically on a given schedule to delete untagged images in the ecr-repo, this will help to limit the count of Images in the ecr-repo.
```
Comment (digitalronin) [inline source/documentation/deploying-an-app/multicontainer-app-deploy.md]: The space between [guide] and [cron-jobs] needs to be removed.

Comment (vijay-veeranki-moj) [reply source/documentation/deploying-an-app/multicontainer-app-deploy.md]: Thanks @digitalronin, fixed it

File: source/documentation/deploying-an-app/multicontainer-app-deploy.md [hunk 2]
```diff
@@ -100,3 +122,10 @@ It should behave in the same way as when you were running it locally via docker-
 [ecr-setup]: tasks.html#creating-an-ecr-repository
 [add-aws-resources]: tasks.html#adding-aws-resources-to-your-environment
 [rds-access-instructions]: https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance#access-outside-the-cluster
+[creating-dashboards]: https://user-guide.cloud-platform.service.justice.gov.uk/tasks.html#creating-dashboards
```
Comment (digitalronin) [inline source/documentation/deploying-an-app/multicontainer-app-deploy.md]: Please change the absolute URLs which point to other parts of the user guide to be relative URLs (i.e. remove "https://user-guide.cloud-platform.service.justice.gov.uk/")

File: source/documentation/deploying-an-app/multicontainer-app-deploy.md [hunk 3]
```diff
@@ -87,6 +87,28 @@ You should be able to view the application in your browser at:
 
 It should behave in the same way as when you were running it locally via docker-compose.
 
+### k8s_additional_resources
```
Comment (digitalronin) [inline source/documentation/deploying-an-app/multicontainer-app-deploy.md]: I'm not sure about this title "k8s_additional_resources" That isn't really what this document is about - monitoring, cron jobs and custom alerts aren't specific to k8s.

How about something like "Further Development" or "Additional Exercises" ?

File: source/documentation/deploying-an-app/multicontainer-app-deploy.md (hunk 2 above)
Comment (vijay-veeranki-moj) [reply source/documentation/deploying-an-app/multicontainer-app-deploy.md]: Thanks @digitalronin updated the URL

File: source/documentation/deploying-an-app/multicontainer-app-deploy.md (hunk 3 above)
Comment (vijay-veeranki-moj) [reply source/documentation/deploying-an-app/multicontainer-app-deploy.md]: Agree with you and renamed it as "Further Development"

**Pull request merged**